helpers/__tests__/paragraphs.test.js

```diff
@@ -0,0 +1,27 @@
+/* global describe, expect, test */
+import * as p from '../paragraph'
+
+describe('summaryHelpers', () => {
+
+  describe('termNotInURL()', () => {
+
+    test('1', () => {
```

Comment:
love to see the tests! this is no big deal, but generally speaking, it's good for the test description to be descriptive so someone other than you running the tests, when it fails, knows what this (failed) test was meant to test for. descriptions generally start with "should". So in this case, instead of "1", it would say:
`test('should find search term in a URL'...`